Gruntfile.js

```diff
@@ -76,7 +76,10 @@ module.exports = function (grunt) {
         stylus: {
             minerva: {
                 files: [{
-                    src: ['<%= plugin.minerva.external %>/stylesheets/main.styl'],
+                    src: [
+                        '<%= plugin.minerva.external %>/stylesheets/main.styl',
+                        '<%= plugin.minerva.root %>/node_modules/colorbrewer/colorbrewer.css'
```

Comment:
I'm building the colorbrewer components into the core minerva source bundles.  The biggest reason I did so is because I didn't want add new files that have to be included everywhere (and I guess I'm lazy).  From what I could tell, there is no equivalent to Girder's `ext.min.js` to put external dependencies into a single place.  Should I do something like that here?